0.75 changelogs (#29402)
microsoft/PowerToys: README.md

Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.

### 0.75 - October 2023 Update

In this release, we focused on stability and improvements.

**Highlights**

>  - New utility: An environment variables editor that also allows configuring profiles that can be enabled/disabled. Thanks [@niels9001](https://github.com/niels9001) for the design and UI work that made this possible!

Reviewer: ```suggestion
 - New utility: An environment variables editor with the functionality to configure profiles that can be enabled/disabled. Thanks [@niels9001](https://github.com/niels9001) for the design and UI work that made this possible!
```
It's not wrong, this just sounds better imo.

 - Settings has a new Dashboard home page, with quick access for enabling modules, short descriptions and activation methods. Thanks [@niels9001](https://github.com/niels9001) for the design and UI work that made this possible!
 - Added a previewer to Peek that hosts File Explorer previewers to support every file type that a machine is currently able to preview. For example, this means that if Microsoft Office handlers are installed, Peek can preview Office files. Thanks [@dillydylann](https://github.com/dillydylann)!

### General

 - Many typo fixes through the projects and documentation. Thanks [@brianteeman](https://github.com/brianteeman)!
 - Refactored and improved the logic across utilities for bringing a window to the foreground after activation.

### Color Picker

>  - Terminate a picker session by using the right mouse button. Thanks [@fredso90](https://github.com/fredso90)!

Reviewer: This is very unclear?

Author: I've rephrased it: "After activating Color Picker, it's now possible to cancel the session by clicking the right mouse button."
Thank you!

### Environment Variables

>  - Added a new utility: An environment variables editor that also allows configuring profiles that can be enabled/disabled. Thanks [@niels9001](https://github.com/niels9001) for the design and UI work that made this possible!

Reviewer: ```suggestion
 - Added a new utility: An environment variables editor that has the functionality to profiles that can be enabled/disabled. Thanks [@niels9001](https://github.com/niels9001) for the design and UI work that made this possible!
```

Same as above

Author: Ended up doing it with a commit, since the suggestion was missing the "configure" word. Thank you ;)

 - Shows in the title bar if it's running as an administrator. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!

### FancyZones

 - Fixed an issue causing context menu pop-ups from some apps to automatically snap to a zone. (This was a hotfix for 0.74)
 - Applied the fix for the context menu pop-ups to the logic that decides which windows can be snapped.

>  - Reworked the "Keep windows in their zones" option to include the work area and turn it on by default, fixing an incompatibility with the Copilot toolbar.

Reviewer: ```suggestion
 - Reworked the "Keep windows in their zones" option to include the work area and turn it on by default, fixing an incompatibility with the Copilot flyout.
```

 - Fixed an issue causing windows to be snapped while moving to a different virtual desktop.

### File Explorer add-ons

 - Fixed an issue blocking some SVG files from being previewed correctly. (This was a hotfix for 0.74)
 - Fixed crashes on invalid files in the STL Thumbnail generator.

### GPO

 - Added a global GPO rule that applies for all utilities unless it's overridden. Thanks [@htcfreek](https://github.com/htcfreek)!
 - Added GPO rules to control which plugins should be enabled/disabled by policy. Thanks [@htcfreek](https://github.com/htcfreek)!

### Image Resizer

 - Fixed wrong .bmp file association in the registry. Thanks [@meitinger](https://github.com/meitinger)!

### Keyboard Manager

>  - Visually distinguish between the Numpad and regular dot characters in the UI.

Reviewer 2: ```suggestion
 - Visually distinguish between the Numpad and regular period characters in the UI.
```

>  - This utility is now disable by default on new installs, since it requires user configuration to do anything useful.

Reviewer: ```suggestion
 - This utility is now disabled by default on new installations, since it requires user configuration to affect keyboard behavior.
```

 - Added options to the shell selection in the Shell plugin and improved the backend code for adding options to plugins. Thanks [@htcfreek](https://github.com/htcfreek)!
 - Added Unix time in milliseconds, fixed negative unix time input and improved error messages in the TimeDate plugin. Thanks [@htcfreek](https://github.com/htcfreek)!
 - The PowerToys plugin allows calling the new Environment Variables utility. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 - Refactored and added support to VSCodium Stable, VSCodium Insider and Remote Tunnels workspaces. Thanks [@eternalphane](https://github.com/eternalphane)!

### Quick Accent

 - Fixed characters that were removed from "All languages" because they were not in any single language. (This was a hotfix for 0.74)
 - Added Asturian characters to the Spanish character set. Thanks [@blakestack](https://github.com/blakestack)!
 - Added Greek characters with tonos. Thanks [@PesBandi](https://github.com/PesBandi)! 

### Registry Preview

 - Fixed a parsing error that crashed the Application. (This was a hotfix for 0.74)

>  - Fixed opening file names with non-ascii characters. Thanks [@randyrants](https://github.com/randyrants)!

Reviewer: ```suggestion
 - Fixed opening file names with non-ASCII characters. Thanks [@randyrants](https://github.com/randyrants)!
```

 - Fixed wrong parsing when the file contained an assignment with spaces around the equals sign. Thanks [@randyrants](https://github.com/randyrants)!
 - Fixed key transversal issues when a key was a substring of a parent key. Thanks [@randyrants](https://github.com/randyrants)!

### Runner

 - Fixed the update notification toast to show an Unicode arrow. Thanks [@TheJoeFin](https://github.com/TheJoeFin)!

### Settings

 - Added a new Dashboard home page, with quick access for enabling modules, short descriptions and activation methods. Thanks [@niels9001](https://github.com/niels9001) for the design and UI work that made this possible!
 - Fixed a typo in the Hosts File Editor page. Thanks [@Deepak-Sangle](https://github.com/Deepak-Sangle)!

>  - Added a lock icon to the flyout listing of all modules when its enabled state is controlled by policy.

Reviewer 3: ```suggestion
 - Added a lock icon to the flyout listing of all modules when its enabled state is controlled by policy.
- The all aps list on the flyout list no all apps even if they are disabled.
```

Author: Thank you for the review, but didn't understand your sentence. Can you please rephrase?
> The all aps list on the flyout list no all apps even if they are disabled.

Reviewer 3: My mistake: Before the change only those Utilities that are not GPO managed were visible in the "All apps" list. No we show all and have the lock icon.

Reviewer 2: Hint: its _now_ 😉

Author: I've rephrased to `The "All apps" list in the flyout will now list all apps even if their enabled state is controlled by policy.` and committed. Hope this makes sense. Thank you for the reviews!

> ### VCM

Reviewer 3: Video Conference Mode

Author: Pushed a commit with "Video Conference Mute". Thank you!

 - Added an option to allow for the toolbar to hide after some time passed. Thanks [@quyenvsp](https://github.com/quyenvsp)!
 - Added an option to select to mute or unmute at startup. Thanks [@quyenvsp](https://github.com/quyenvsp)!
 - Fixed an issue causing a cascade of mute/unmute triggers.

### Documentation

>  - Removed the Group Policy documentation from the repo and linked to the published documentation on learn.microsoft.com instead.

Reviewer 3: ```suggestion
 - Updated the Group Policy documentation on learn.microsoft.com, removed the Group Policy documentation from the repository and linked to the published documentation on learn.microsoft.com instead.
```
